eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal

```
@@ -65,17 +69,65 @@ library
         plutus-ledger-api:{plutus-ledger-api, plutus-ledger-api-testlib} ^>=1.5,
         QuickCheck,
         random,
+        serialise,
         small-steps >=1.0,
         small-steps-test,
+        time,
         transformers
 
+executable gen-golden
+    main-is:
+        Test/Cardano/Ledger/Alonzo/Translation/GenerateGoldenFileMain.hs
```

Main file is usually never buried that deep:
```suggestion
        GenerateGoldenFileMain.hs
```